compare-evidence-strings/compare.sh

```diff
@@ -7,30 +7,25 @@
 echo "Defining functions"
 
 # A function to sort keys in the evidence strings. This makes them more readable and helps comparison through word diff.
-# Also removes several version- and date-related fields which are changed frequently, but do not reflect actual change:
-# * validated_aginst_schema_version
-# * date_asserted
 # The two arguments are input and output JSON files.
 sort_keys () {
   jq -c -S "." <"$1" \
-    | sed -e 's|,"validated_against_schema_version":"[0-9.]*"||g' \
-    | sed -e 's|"date_asserted":".\{19\}",||g' \
```

Comment:
I presume these are the ``unique_identifying_fields`` that Kirill [mentioned](https://github.com/EBIvariation/eva-opentargets/issues/201#issue-827323123)?

Reply:
Okay, it seems they are not the ones he referred to. Is there another mention on why these 2 fields that are said to vary a lot disappear in JSON Schema V2?

Reply:
I guess they were just deemed unnecessary by OT...  the full discussion on the schema migration is in #189

Reply:
Okay that's the answer I was looking for. I can see that they simply made them disappear 👍